service/src/test/java/bio/terra/workspace/common/logging/WorkspaceActivityLogHookTest.java

```diff
@@ -428,5 +429,13 @@ public String prettyStepState() {
     public String flightDesc() {
       return null;
     }
+
+    @Override
+    public ProgressMeter getProgressMeter(String name) {
+      return null;
+    }
+
+    @Override
+    public void setProgressMeter(String name, long v1, long v2) throws InterruptedException {}
```

Comment:
Mostly unrelated, these got pulled in with the new stairway version from the updated TCL